tools/init_databases.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+# import required module
+import os
+import sqlite3
+import argparse
+
+
+# this helper script creates DBs from SQL schema files
+
+def process_file(path: os.path, dst_directory: os.path) -> None:
+    file_name = os.path.basename(path)
+    print(file_name)
```

Comment:
clear out print statements or use `logging` package with `DEBUG` level if this script is meant to use on CI

Reply:
added logging

Reply:
Ok